core/src/main/java/com/linecorp/armeria/internal/server/annotation/KotlinUtil.java

```diff
@@ -51,6 +55,9 @@
     @Nullable
     private static final Method IS_RETURN_TYPE_UNIT;
 
+    @Nullable
+    private static final Method K_FUNCTION_RETURN_TYPE;
```

Comment:
Could we convert `Method` to `MethodHandle` with `MethodHandles.lookup().unreflect()`?
The speed of `MethodHandle` is comparable to the normal method invocation.

Reply:
I thought the original author of this class intentionally used `Method` for those methods which will only be invoked on server startup and never used later?

Reply:
Indeed. Let's keep it as it is.